conf.pri.mingw

```diff
@@ -0,0 +1,43 @@
+# Adapt these paths
+# Point these to the include folders
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_64_0)
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_1_1/include)
```

Comment:
nitpick though, `libtorrent` in place of `RC_1_1` is more clear.
Also I would use `boost` in place of `boost_1_64_0`.

Reply:
If we install under same base dir we have:
`C:/qBittorrent/include` <--- libtorrent and other "normal libs"
`C:/qBittorrent/include/boost_1_64_0` <---- boost includes

But yeah, removing version info is better imo

Reply:
Ok, good idea. Done.